oci/tests/integration/testapp/main.go

```diff
@@ -77,7 +79,12 @@ func main() {
 		panic(err)
 	}
 
-	auth, err = login.NewManager().Login(ctx, loginURL, ref, opts)
+	if *oidcLogin && *registry != "" {
```

Comment:
> Wouldn't this result in the Registry + repo together in one string case to call the old login

This what I am trying to avoid, loginURL == registry (when the registry flag is set).
When it isn't set, loginURL == registry + repo (which is the wrong URL to pass to the oidclogin)